modules/phpfpm/client.go

```diff
@@ -40,34 +43,69 @@ func (rd *requestDuration) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
-type client struct {
+type client interface {
+	getStatus() (*status, error)
+
+}
+
+type httpClient struct {
+	module.Base
```

Comment:
both `httpClient` and `socketClient` should implement `client` only, then dont need to have `module.Base`

`Phpfpm` should have `client` interface

```go
	Phpfpm struct {
		module.Base
		Config `yaml:",inline"`

        client client
	}
```

and we need to initialize appropriate client base on provided configuration in `Init()`